src/main/core/Controller/APINew/Log/SecurityLogController.php

```diff
@@ -34,4 +44,27 @@ public function listAction(Request $request): JsonResponse
             []
         ));
     }
+
+    /**
+     * @Route("/list/current", name="apiv2_logs_security_list_current", methods={"GET"})
+     */
+    public function UserLogSecurtiyAction(Request $request): JsonResponse
+    {
+        if (!$this->authorization->isGranted('IS_AUTHENTICATED_FULLY')) {
+            throw new AccessDeniedException();
+        }
+
+        $user = $this->tokenStorage->getToken()->getUser();
+
+        $query['hiddenFilters'] = [
+            'user' => $user->getUuid(),
+            'email' => $user->getEmail(),
```

Comment:
You don't need to search on email too.

Reply:
When someone tries to log in and the password does not match, it is the email address that is recorded in the message
I added this to retrieve logs of failed authentications

Reply:
If the email is known, you'll retrieve the correct user in your listener and can use the `user` filter. If the email is not known, the log will only be shown in admin because we can not link it to any user.

Also note, we always do an `AND` between filters.